src/pages/instances/InstanceStatusIcon.tsx

```diff
@@ -21,6 +21,7 @@ const InstanceStatusIcon: FC<Props> = ({ instance }) => {
         Ready: "status-waiting-small",
         Running: "status-succeeded-small",
         Stopped: "status-queued-small",
+        Migrating: "status-in-progress-small",
```

Comment:
I don't think we ever get a Migrating status from the backend, do we?
```suggestion
```

Reply:
aah yes that's correct, will remove